lib/brew.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import os
+from shutil import which
+
+from devenv.lib import fs
+from devenv.lib import proc
+
+repo_path = "/opt/homebrew"
+
+
+def install() -> None:
+    if which("brew") == f"{repo_path}/bin/brew":
+        return
+
+    shellrc = fs.shellrc()
+    print("You may be prompted for your password to install homebrew.")
+    while True:
```

Comment:
Should we limit how many times this runs?

Reply:
No, people can ^C out at any time.